dev/com.ibm.ws.security.oauth/resources/com/ibm/ws/security/oauth20/internal/resources/OAuthMessages.nlsprops

```diff
@@ -134,7 +138,7 @@ OAUTH_CLIENT_REGISTRATION_MALFORMED_REQUEST.explanation=The JSON in the request
 OAUTH_CLIENT_REGISTRATION_MALFORMED_REQUEST.useraction=Review the API documentation for this service, and retry the request.
 
 OAUTH_CLIENT_REGISTRATION_CLIENTID_EXISTS=CWWKS1429E: Client id {0} already exists.
-OAUTH_CLIENT_REGISTRATION_CLIENTID_EXISTS.explanation=The operation failed as the client id {0} already exists.  
+OAUTH_CLIENT_REGISTRATION_CLIENTID_EXISTS.explanation=The operation failed as the client id already exists.  
```

Comment:
Change
"The operation failed as the client id already exists."
To
"The operation failed because the client id exists."